src/main/webapp/app/shared/link-preview/components/link-preview/link-preview.component.scss

```diff
@@ -0,0 +1,85 @@
+.loading-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+}
+
+.loading-spinner {
+    display: flex;
+    align-items: center;
+}
+
+.spinner {
+    border: 4px solid #f3f3f3; /* Light gray */
```

Comment:
Why don't you use some of the predefined colors as described here? https://docs.artemis.cit.tum.de/dev/guidelines/client-design/

Reply:
changed to `var(--secondary)` for the light gray color 👍